tests/conftest.py

```diff
@@ -4,11 +4,12 @@
 
 import pytest
 
+from kpops.utils.environment import ENV, Environment
 from kpops.utils.yaml import load_yaml_file
 
 
 @pytest.fixture()
-def mock_env() -> Iterator[os._Environ[str]]:
+def mock_os_env() -> Iterator[os._Environ[str]]:
```

Comment:
technically not used anymore, but I'd like to keep this around in case we need it again. It's such a handy snippet, already stole it for other internal projects :)